util/file_test.go

```diff
@@ -95,14 +106,17 @@ func TestIsDirReal(t *testing.T) {
 	assert.Equal(t, "Path is not a directory", err.Error())
 	assert.False(t, ok)
 
-	symLinkPath := TempPath("", "TestIsDirReal")
-	err = os.Symlink(os.TempDir(), symLinkPath)
-	defer RemoveFileAtPath(symLinkPath)
-	assert.NoError(t, err)
-	ok, err = IsDirReal(symLinkPath)
-	assert.Error(t, err)
-	assert.Equal(t, "Path is a symlink", err.Error())
-	assert.False(t, ok)
+	// Windows requires privileges to create symbolic links
```

Comment:
is this true? in our repo we use mklink /j which (i don't think) requires elevated priveledges